tasks/lint.yaml

```diff
@@ -11,3 +11,9 @@ tasks:
       # requires yamllint to be installed either from pip or brew
       - description: Lint YAML files
         cmd: yamllint . -c .yamllint --no-warnings
+
+  - name: oscal
+    description: Run linting checks on OSCAL
+    actions:
+      - description: Lint OSCAL files
+        cmd: lula tools lint -f $OSCALFILES
```

Comment:
This should probably be defined as a variable so that Maru can pick it up with `--set` and so that there is a `default` without needing to set an env var (i.e. if you ran this manually locally.)

Reply:
(I know we mentioned inputs before but I wouldn't worry about those for this since this is intended to be run directly)